### PR TITLE
[skip-ci][DF]Fixes of the df015 tutorial documentation

### PR DESCRIPTION
**Pull request by j-mathe**

### CODE REVIEW

File: tutorials/dataframe/df015_LazyDataSource.C [hunk 1]
```diff
@@ -24,15 +24,15 @@ int df015_LazyDataSource()
    using namespace ROOT::RDF;
 
    // Let's first create a RDF that will read from the CSV file.
-   // See the tutorial relative to CSV data sources for more details!
+   // See the tutorial on CSV data sources for more details!
```
Comment (eguiraud) [inline tutorials/dataframe/df015_LazyDataSource.C:27]: can we make this a link to the actual tutorial?

Comment (j-mathe) [reply tutorials/dataframe/df015_LazyDataSource.C]: I tried to make a link similar to the other links in the tutorials but I didn't find out how to check whether it's "clickable" when opened as html. Do you know how to do that? If it works, I can merge the commits.

Comment (eguiraud) [reply tutorials/dataframe/df015_LazyDataSource.C]: that's ok, you can check the newly generated HTML tomorrow morning to make sure it displays correctly

Comment (j-mathe) [reply tutorials/dataframe/df015_LazyDataSource.C]: Where is this HTML?

Comment (eguiraud) [reply tutorials/dataframe/df015_LazyDataSource.C]: Sorry I meant it's going to be at https://root.cern/doc/master/group__tutorial__dataframe.html the day after we merge the PR :)

**Review by eguiraud — APPROVED** [review]
:tada:

**Pull request merged**